src/index.js

```diff
@@ -0,0 +1,52 @@
+let tempValue = 32;
+
+const changeTemperature = (currentTemp) => {
+    let tempValue = document.querySelector('#tempValue');
+    tempValue.textContent = currentTemp + ` °F`;
+    let garden = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+    if (currentTemp >= 80) {
+        garden = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
```

Comment:
To achieve the dynamic color changing, you can use this same conditional to add a className to `tempValue` so that you can style it differently depending on the temp.

```suggestion
        garden = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'
        tempValue.className = 'red'
```

#index.css
```
.red {
   color = red
}